dashboard/app/controllers/scripts_controller.rb

```diff
@@ -112,11 +112,10 @@ def edit
       beta: beta,
       betaWarning: beta_warning,
       levelKeyList: beta && Level.key_list,
-      stageLevelData: @script_file,
+      lessonLevelData: @script_file,
```

Comment:
Is it ok to make this change over or should i keep stageLevelData for a little bit?